backend/src/game/dice_handler.py

```diff
@@ -0,0 +1,24 @@
+import backend.src.game.dice as dice
+import backend.src.game.constants as constants
+
+
+class DiceHandler(object):
+    dice_values = list()
```

Comment:
Use a list literal instead of this. It's the idiomatic way and it's also faster: https://softwareengineering.stackexchange.com/questions/319547/literals-versus-instantiating-by-name-lists-and-dicts-in-python

Reply:
Again, thanks for including the references to read, it's been really helpful!